client/rpc-api/src/state/mod.rs

```diff
@@ -39,10 +39,20 @@ pub trait StateApi<Hash> {
 	#[rpc(name = "state_call", alias("state_callAt"))]
 	fn call(&self, name: String, bytes: Bytes, hash: Option<Hash>) -> FutureResult<Bytes>;
 
-	/// Returns the keys with prefix, leave empty to get all the keys
+	/// Returns the keys with prefix, leave empty to get all the keys.
```

Comment:
Shall we deprecate this method then? I imagine adding deprecation notice in the code and printing a warning when the method is used (but at most once a minute or so).

Reply:
printing a warning on server wouldn't be helpful unless we modify the response to indicate this method is deprecated. better do it on SDK side?